pkg/server/plugin/upstreamca/awssecret/awslib_fake.go

```diff
@@ -0,0 +1,61 @@
+package awssecret
+
+import (
+	"io/ioutil"
+	"net/http"
+
+	"github.com/aws/aws-sdk-go/aws/request"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+	"github.com/aws/aws-sdk-go/service/secretsmanager/secretsmanageriface"
+	"github.com/stretchr/testify/suite"
+)
+
+type fakeSecretsManagerClient struct {
+	suite.Suite
+	secretsmanageriface.SecretsManagerAPI
+
+	mockStorage map[string]string
```

Comment:
nit: since this is in the context the fake, i'd recommend simply naming it `storage` or `secrets`.